### ISSUE TITLE
Possibility to finetune language model only (unsupervised)

### ISSUE THREAD

**Issue by thinline72** [IndicoDataSolutions/finetune#135]
Hi all,

Is it possible to finetune in unsupervised way, like language model firstly/only?

I have a lot of unlabelled data and just a hundreds of labelled examples, so I'd like to firstly finetune LM in unsupervised way and then finetune on specific supervised task. Such process was described in ULMFit paper.

It might be also pretty useful for getting better deep representation if you don't have labelled examples at all.

**Comment by madisonmay** [comment]
Yup!  This is absolutely possible -- just a matter of calling `model.fit(text)` without providing labels.  See issue #56.

I would expect this unsupervised finetuning process to be useful but haven't had the opportunity to run substantive experiments on this front, so definitely let us know how it goes!

**Comment by madisonmay** [comment]
Yup, your understanding is correct!

**Comment by thinline72** [comment]
Great, thank you a lot for you help!
